src/transaction_manager/transaction_manager.hpp

```diff
@@ -75,15 +63,13 @@ class TransactionManager : public std::enable_shared_from_this<TransactionManage
    * @brief Inserts batch of unverified broadcasted transactions to unverified queue and db
    *
    * @note Some of the transactions might be already processed -> they are not processed and inserted again
-   * @param raw_trxs transactions to be processed
+   * @param txs transactions to be processed
    * @return number of successfully inserted unseen transactions
    */
-  uint32_t insertBroadcastedTransactions(const std::vector<taraxa::bytes> &raw_trxs);
-
-  std::pair<bool, std::string> verifyTransaction(Transaction const &trx) const;
+  uint32_t insertBroadcastedTransactions(const std::vector<Transaction> &txs);
 
   std::unordered_map<trx_hash_t, Transaction> getVerifiedTrxSnapShot() const;
-  std::vector<taraxa::bytes> getNewVerifiedTrxSnapShotSerialized();
+  std::vector<Transaction> getNewVerifiedTrxSnapShotSerialized();
```

Comment:
this could be also `const` function

Reply:
it cant as it calls method on trx_queue which cannot be const